extensions/BugModal/web/bug_modal.js

```diff
@@ -918,9 +918,9 @@ $(function() {
                 query_string += `&bug_id=${BUGZILLA.bug_id}`;
             }
             if (has_needinfo_from) {
-                query_string += "&note=Needinfo requested by " + encodeURIComponent(has_needinfo_from);
+                query_string += "&note=Needinfo+requested+by+" + encodeURIComponent(has_needinfo_from);
             }
-            window.location.replace(`${BUGZILLA.config.basepath}userprefs.cgi?` + query_string);
+            window.location = `${BUGZILLA.config.basepath}userprefs.cgi?` + query_string;
```

Comment:
Using string concatenation _and_ interpolation isn't ideal; use one or the other (interpolation preferred).